src/Microsoft.AspNetCore.Sockets.Client/LongPollingTransport.cs

```diff
@@ -28,10 +29,14 @@ public class LongPollingTransport : ITransport
 
         public Task Running { get; private set; }
 
+        public LongPollingTransport(HttpClient httpClient)
```

Comment:
not sure if we need these overloads for transports. Connection will always pass a factory and the users are not really expected to create these.